README.md

```diff
@@ -60,6 +60,10 @@ tabnine:setup({
 })
 ```
 
+## Log in to your account
```

Comment:
Please change to something which does not sound like you *have* to log in to your account.

Reply:
How does "Activate your pro account" sounds to you?

Reply:
I don't like the `pro` part....
Maybe `Configure Tabnine or Login to Your Account` ???

Reply:
done